THIRD-PARTY-NOTICES.TXT

```diff
@@ -69,34 +69,21 @@ written authorization of the copyright holder.
 License notice for Zlib
 -----------------------
 
+https://github.com/zlib-ng/zlib-ng/blob/develop/LICENSE.md
 https://github.com/madler/zlib
```

Comment:
Do we still need the references to madler/zlib? Or do those come implicitly as part of the zlib-ng license? Mainly I'm wondering if we're still relying on anything from madler/zlib anywhere.

Reply:
Ah, I can remove it. Thanks for noticing.

The madler changes are implicit in zlib-ng. Plus the zlib-ng license is the same as the one in madler/zlib.